frontend/src/components/design-system/containers/CardComponent.vue

```diff
@@ -0,0 +1,90 @@
+<script setup lang="ts">
+import { computed } from 'vue';
+
+export interface Props {
+  radius?: 'small' | 'large';
+  corner: 'top-left' | 'top-right' | 'bottom-left' | 'bottom-right';
+  border?: boolean | 'strong';
+  shadow?: boolean;
+  padding?: 'small' | 'large';
+}
+
+const props = withDefaults(defineProps<Props>(), {
+  radius: 'small',
+  border: false,
+  shadow: false,
+  padding: 'small',
+});
+
+const classes = computed(() => ({
+  [`radius--${props.radius}`]: props.radius,
+  [`corner--${props.corner}`]: props.corner,
+  border: props.border,
+  'border--strong': props.border === 'strong',
+  shadow: props.shadow,
+  [`padding--${props.padding}`]: props.padding,
+}));
+</script>
+
+<template>
+  <div :class="classes">
+    <slot />
+  </div>
+</template>
+
+<style scoped lang="scss">
+div {
+  background-color: rgb(var(--color-neutral-bg-container));
+}
+
+.radius {
+  &--small {
+    border-radius: var(--radius-small);
+  }
+  &--large {
+    border-radius: var(--radius-large);
+  }
+}
+
+.corner {
+  &--top-left {
+    border-top-left-radius: 0;
+  }
+  &--top-right {
+    border-top-right-radius: 0;
+  }
+  &--bottom-left {
+    border-bottom-left-radius: 0;
+  }
+  &--bottom-right {
+    border-bottom-right-radius: 0;
+  }
+}
+
+.border {
+  border-style: solid;
+  border-width: var(--spacing-pico);
+  border-color: rgb(var(--color-neutral-border));
+
+  &--strong {
+    border-color: rgb(var(--color-neutral-border-strong));
+  }
+}
+
+.shadow {
+  box-shadow: 6px 8px 0px 0px
+    rgba(var(--color-neutral-shadow), var(--transparency-20));
+}
+
+.padding {
+  &--small {
+    padding: calc(var(--spacing-small) - var(--spacing-pico));
```

Comment:
Proč se tu odečítá to pico? To tu jedeme pixel perfect? :D

Reply:
Ano! :D Pico je border, takže aby nám to hezky sedělo do gridu (ono jako u tý card je to asi jedno, ale třeba tlačítko už to rozbíjí slušně).